source/_components/hangouts.markdown

```diff
@@ -0,0 +1,128 @@
+---
+layout: page
+title: "Google Hangouts"
+description: "Hangouts chatbot support"
+date: 2018-08-18 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: hangouts.png
+ha_category: Hub
+ha_release: 0.77
+---
+
+This component allows you to send messages to [Google Hangouts](http://hangouts.google.com) conversations, as well as to react to messages in conversations. Reacting to commands is accomplished by firing an event when one of the configured commands is triggered.
+
+```yaml
+# Example configuration.yaml entry
+hangouts:
+  email: my_google_user@gmail.com
```

Comment:
Email address should be in quotes - as mentioned below

Reply:
i removed both, they are not used anymore, instead of this i added a part for the config flow.